greenpeace/planet4-child-theme-australiapacific
# Reduce line-height in buttons

# Pull Request Overview
This sets button line height to be more reasonable, because before it was very big. Reducing the line height meant that the spacing between the text and the edge of the button decreased, so I've increased padding to get that back to what it was. 

Fixes [this ticket](https://www.notion.so/greenpeaceaustraliapacific/P4-Line-height-on-buttons-054cbfe964cc4289868df05e2bfeb112?pvs=4)

# 📷 Screenshots

| Before | After |
|:-:|:-:|
|![image](https://github.com/greenpeace/planet4-child-theme-australiapacific/assets/13824257/26b690f7-ceb7-4148-90e3-a1408022c70a)|![image](https://github.com/greenpeace/planet4-child-theme-australiapacific/assets/13824257/89b00a81-5491-461b-aed0-064a5a35a1eb)|

# QA / Test steps

1. Create a blank post
2. Add a 2 column layout
3. Inside one column, add a button, with long text like "i am a button with some really long text it goes on an on I swear"
5. Observe the before/after from `main` to `feature/fix-button-line-height` branches
